.owners.yml

```diff
@@ -0,0 +1,10 @@
+assign:
+  strategy:
+    # random
+    daily-shift-based
+  assignees:
+    - MeowZheng
+    - MengzhangLI
+    - linfangjian01
+    - RockeyCoss
+    # - chenlin
```

Comment:
This means 4-day iteration including weekend, so I think the regular rotation would become a mess. Could we add more people or only set workday?